app/controllers/ncr/work_orders_controller.rb

```diff
@@ -14,7 +14,9 @@ def create
     end
 
     def edit
-      if self.proposal.approved?
+       @client_data_instance.approving_official_email = @client_data_instance.approvers.first.email_address
```

Comment:
Why not `Ncr::WorkOrderValueNormalizer.new(@client_data_instance).run` which already has the approving_official_email logic (amongst others)?